cmd/rollkit/commands/intercept.go

```diff
@@ -65,6 +65,28 @@ readTOML:
 	return true, runEntrypoint(&rollkitConfig, flags)
 }
 
+func buildEntrypoint(rootDir, entrypointSourceFile string, forceRebuild bool) (string, error) {
+	// The entrypoint binary file is always in the same directory as the rollkit.toml file.
+	entrypointBinaryFile := filepath.Join(rootDir, rollupBinEntrypoint)
```

Comment:
[non-blocking] As a general point on naming, this should be `entrypointBinaryFilePath` since it is the file path to the file, not a File object.

Reply:
yeah, make sense